Add info about using Projects and Boards (#1707)
XRPLF/xrpl-dev-portal: README.md

If you added a new language for the first time, you need to make sure its target definition (in the `dactyl-config.yml` file) has the MO file in the `locale_file` field.

After that, next time you build the site using Dactyl it should pull the updated translations!

## Issues, Projects, and Project Boards

> Use GitHub Issues to report bugs, feature requests, and suggestions. 

Reviewer: Maybe clarify that the xrpl-dev-portal repo is for bugs, feature requests, and suggestions _for the docs or website specifically_. It's not uncommon for us to get people reporting issues with the rippled server or client libraries here.

Reviewer 2: For those issues, we could point people to https://github.com/XRPLF and/or the rippled, xrpl.js, xrpl-py, et al. repos

Author: Incorporated feedback

If you are a contributor, use GitHub Projects and Project Boards to plan and track updates to xrpl.org. 

### Project Board `xrpl-docs`

> The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages. 

Reviewer: I don't think we need to treat "Board" and "Issues" as proper nouns here. But also, I'm not sure what "The board only reflects the status of the issues" is supposed to mean—as opposed to what? It feels like we can just cut that section out and not lose much.

```suggestion
The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. Contributors must update the status of an issue as it progresses through different stages. 
```

> The `xrpl-docs` board has six columns based on the status of Issues in this repository:

Reviewer: ```suggestion
The `xrpl-docs` board has six columns based on the status of issues in this repository:
```

> * **No Status**: Lists new or existing issues which require their Status to be updated.  

Reviewer: ```suggestion
* **No Status**: New or existing issues that no one has triaged yet.
```

> * **Backlog**: Lists issues that have been triaged by repository maintainers or contributors. Ensure that the GitHub Issue contains actionable and helpful information for a contributor to work on addressing the issue. 

Reviewer: ```suggestion
* **Backlog**: Issues that represent tasks to be done eventually. They should contain actionable and helpful information for a contributor to work on addressing the issue.
```

> * **Planned**: Lists issues that have an assignee and are planned to be addressed in the near future, like 2-4 weeks. 

Reviewer: ```suggestion
* **Planned**: Issues with assignees who plan to address them in the near future, like 2-4 weeks.
```

> * **In Progress**: Lists issues that are in progress. 

Reviewer: ```suggestion
* **In Progress**: Issues that a contributor is actively working on.
```

> * **In Review**: Lists issues that are being reviewed. 

Reviewer: ```suggestion
* **In Review**: Issues with a proposed fix that is currently being reviewed. These should be associated with an open pull request.
```

> * **Done**: Lists issues that have been completed and the related content updates have been merged. 

Reviewer: ```suggestion
* **Done**: Issues that have been completed, whose related content updates have been merged. 
```
